sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/BasicDigitalTwinPropertyMetadata.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.OffsetDateTime;
+
+/**
+ * Contains metadata about changes on properties on a digital twin or component.
+ */
+public class BasicDigitalTwinPropertyMetadata {
+    @JsonProperty(value = DigitalTwinsJsonPropertyNames.METADATA_PROPERTY_LAST_UPDATE_TIME, required = true)
+    private OffsetDateTime lastUpdatedOn;
```

Comment:
I know we decided to use `lastUpdatedOn`, I just wanted to say I think `lastUpdatedTime` might have been a better property name. goodbye

Reply:
Sadly this is the name the service choose, so I'm not looking to change it.